src/main/java/redis/clients/jedis/exceptions/JedisBroadcastException.java

```diff
@@ -0,0 +1,52 @@
+package redis.clients.jedis.exceptions;
+
+import java.util.HashMap;
+import java.util.Map;
+import redis.clients.jedis.HostAndPort;
+
+/**
+ * Note: This exception extends {@link JedisDataException} just so existing applications catching
+ * JedisDataException do not get broken.
+ */
+public class JedisBroadcastException extends JedisDataException {
+
+  private static final String BROADCAST_ERROR_MESSAGE = "<a generic broadcast error message goes here.>";
```

Comment:
```"<a generic broadcast error message goes here.>"```

I think we need one

Reply:
@chayim Please give me one

Reply:
@uglide will care more than me about text. I suggest
```A failure occurred broadcasting the command```